src/renderer/redux/reducers/latestVersionNames.reducer.ts

```diff
@@ -0,0 +1,17 @@
+import * as actionTypes from '../actionTypes';
+import { SetModAndTrackLatestVersionName } from '../types';
+
+export type ModAndTrackLatestVersionNames = { modKey: string, trackKey: string, name: string }[]
```

Comment:
All Redux types should be stored in `../types`

Reply:
Agreed. Imports aren't nice looking like this: `import { ModAndTrackLatestVersionNames } from "renderer/redux/reducers/latestVersionNames.reducer";`